internal/Employee.go

```diff
@@ -1,6 +1,16 @@
 package internal
 
-type Employee struct {
+type Employee interface{}
+
+type EmployeeResponse struct {
+	Employee
+	FirstName  string    `json:"firstName"`
+	LasName    string    `json:"lasName"`
+	PositionID *Position `json:"position"`
```

Comment:
Here should being used ID of position entity, not the pointer to the struct.

Reply:
+